StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentViewController.swift

```diff
@@ -111,6 +115,19 @@ class ConsentViewController: UIViewController {
         // this fixes an issue where presenting a UIViewController
         // on top of ConsentViewController would stop the dot animation
         consentLogoView?.animateDots()
+
+        NotificationCenter.default.addObserver(
```

Comment:
this likely doesn't matter too much, but wouldn't viewWillAppear potentially get called multiple times, so a potentially more appropriate place to "subscribe only once" is viewDidLoad (or something of the like)?

Reply:
Good point! I'll move this